modules/nf-core/chopper/main.nf

```diff
@@ -0,0 +1,36 @@
+process CHOPPER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::chopper=0.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/chopper:0.3.0--hd03093a_0':
+        'quay.io/biocontainers/chopper:0.3.0--hd03093a_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta)                 , emit: meta
+    path "versions.yml"             , emit: versions
+    tuple path("*.filtered.fastq.gz")  , emit: fastq
```

Comment:
```suggestion
    tuple val(meta)                   , emit: meta
    path "versions.yml"               , emit: versions
    tuple path("*.filtered.fastq.gz") , emit: fastq
```

Harshil Alignment™️

Reply:
should be fixed in new commit
also changed it to

    tuple val(meta), path("${prefix}.fastq.gz") , emit: fastq
    path "versions.yml"                         , emit: versions

(hopefully) in accordance with point six of the [input/output options](https://nf-co.re/docs/contributing/modules#inputoutput-options)